Formula/boost-python.rb

```diff
@@ -78,12 +56,10 @@ def install
         boost::python::def("greet", greet);
       }
     EOS
-    Language::Python.each_python(build) do |python, _|
-      pyflags = (`#{python}-config --includes`.strip + " " +
-                 `#{python}-config --ldflags`.strip).split(" ")
-      system ENV.cxx, "-shared", "hello.cpp", "-L#{lib}", "-lboost_#{python}", "-o", "hello.so", *pyflags
-      output = `#{python} -c "from __future__ import print_function; import hello; print(hello.greet())"`
-      assert_match "Hello, world!", output
-    end
+    pyflags = (`python-config --includes`.strip + " " +
```

Comment:
I know this code didn't originate from you, but seems like a decent time to kill off some of the backticks in favour of `Utils.popen_read`.

Reply:
Done